### ISSUE TITLE
App is not working - FireOS TV

### ISSUE THREAD

**Issue by github-gitlab** [localsend/localsend#1686]
### Describe the bug

App seems to have stopped working on FireOS. It is crashing immediately after opening. I tried to clear data and cache first but that didn't work so I uninstalled and redownloaded the app but to no avail.

### To reproduce

Just open the app to reproduce the crash 

### Expected behavior

App should open and work as before.

**Comment by Tienisto** [comment]
Can you somehow try out an older version of LocalSend? Maybe 1.13? I remember that it worked some time ago. If you are able to tell us since when the app stopped working, then we can hopefully fix this problem.

**Comment by github-gitlab** [comment]
Yes, it worked perfectly until yesterday. I have been using it since long and never encountered such an issue. I am currently running the latest version and I am guessing it is due to that. Let me try 1.13 and get back to you.

    On Friday, August 23, 2024 at 12:46:03 AM GMT+5:30, Tien Do Nam ***@***.***> wrote:  
 
 


Can you somehow try out an older version of LocalSend? Maybe 1.13? I remember that it worked some time ago. If you are able to tell us since when the app stopped working, then we can hopefully fix this problem.

**Comment by github-gitlab** [comment]
I tried 1.13.0, 1.14.0 and 1.15.0; all worked except the latest 1.15.4 available on app store.
